changelog

```diff
@@ -1,3 +1,400 @@
+        * Reduce chassic cost: 1090000 -> 836000
+        * Increased cargo space: 10 -> 15
+        * Removed the Cooling Ducts and use a cheaper generator.
+      * Reduce the 0.9.15 afterburner buff. (@Amazinite)
+        * Reduced thrust by 20-30% and increased fuel usage by 20-30% on all afterburners.
+        * For reference, in 0.9.15, afterburners gained a 50% increase in thrust, 70-80% decrease in fuel usage, and 33% decrease in size.
+    * Graphics:
+      * Added sprites for the Remnant Swan. (@Saugia)
+      * Remodelled Korath Thermal Repeater sprites. (@Daeridanii1)
+      * Remodelled the Android sprite. (@Anarchist2)
+      * Added sprites for a Republic Navy Battleship and a spinal weapon for it. (@GefullteTaubenbrust2)
+      * Reduced the scaling of Hai stations from 0.5 to 0.4. (@Azure3141)
+      * New landscape images for Earth, New China, Chiron, and Solace. (@bene-dictator)
+      * Remodelled the Hai railgun, railgun slug, and railgun slug rack. (@1010todd)
+      * Expanded and improved the Milky Way galaxy sprite. (@Azure3141)
+      * Remodelled the Enforcer Riot Staff and Enforcer Confrontation Gear. (@1010todd)
+      * Added a brown haze. (@RisingLeaf)
+      * Sprites for Hai optical jamming outfits. (@beccabunny)
+      * Added a sprite for a proto-planetary disk. (@RisingLeaf)
+    * Sounds:
+      * Reduced the volumes of the Particle and Plasma Cannon sounds. (@Saugia)
+      * A new sound for the Ion Hail Turret. (@Saugia)
+      * A unique sound for the Moonbeam and Moonbeam Turret. (@Saugia)
+      * Unique sounds for the Ember Tear. (@Saugia)
+    * Other:
+      * Moved Unfettered missions and jobs to their own files. (@Hurleveur)
+      * Updated credits. (@Zitchas)
+        * Added Lia Gerty and MasterOfGrey as faction authors.
+        * Added Saugia and Anarchist2 as devs.
+      * Reduced the speed of minable asteroids in Coalition space. (@Azure3141)
+      * Inform the player if they lose their Jump Drive while it is still necessary in the Free Worlds story. (@Anarchist2)
+      * Improved descriptions of various planets: Longjump, Buccaneer Bay, Poisonwood, Bourne, Millrace, and Solace. (@bene-dictator)
+      * Added a new category to the shipyard: Superheavy. (@Saugia)
+        * Also moved the Ikatila'ej, Korsmmanath A'awoj, Heron, and Emerald Sword to this category.
+      * Danforth's ship now also have the "timid" personality in "Wanderers: Alpha Surveillance G". (@Quantumshark)
+      * Add some diversity to the sale of Hai ships. (@MasterOfGrey)
+      * Moved the Cloaking Device is now to the "Unique" outfit category. (@czartrak)
+      * The descriptions of the Heliarch reactors now indicate that they used anti-matter catalyzed fusion technology. (@Arachi-Lover)
+      * Governments that reuse colors now refer to named "stock" colors instead of providing their own color literals. (@warp-core)
+      * Added a reference to the player's manual to the credits. (@hmglasgow)
```

Comment:
```suggestion
      * Added a reference to the credits in the player's manual. (@hmglasgow)
```

Reply:
If that's what the intent is meant to be, then yes. Way I read the initial I thought it was the way around I had it :)